app/assets/javascripts/application.js

```diff
@@ -85,6 +85,7 @@ $(function() {
   $('a[href*=#]:not([href=#])').click(function() {
     if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
       var target = $(this.hash);
+      history.pushState({}, "Community Favorites", this.hash);
```

Comment:
No matter how little code you contribute, hound will find something wrong with it.